superset/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx

```diff
@@ -122,6 +143,7 @@ function mapStateToProps(state) {
     alert: state.controlPanelAlert,
     isDatasourceMetaLoading: state.isDatasourceMetaLoading,
     fields: state.fields,
+    exploreState: state,
```

Comment:
I think it's good to treat the props individually. I wanted `getProps` to receive the whole state so that it can act like some a `mapStateToProps` for individual fields.

I could rename `getProps` to `mapStateToProps` if that adds clarity.